docs/developers/NewToGluten.md

```diff
@@ -84,25 +83,24 @@ root@xx2:~ps ux | grep intellij
 root@xx2:kill -9 <pid>
 ```
 
-And then restart idea
+And then restart idea.
 
 ## Windows/Mac intellij remote debug
 
-If you have Ultimate intellij, you can try to debug remote
+If you have Ultimate intellij, you can try to debug remotely.
 
 ## Set up gluten project
 
-- Make sure you have compiled gluten
-- Load the gluten by File->Open, select <gluten_home/pom.xml>
-- Activate your profiles such as <backends-velox>, and Reload Maven Project, you will find all your need modules have been activated
-- Create breakpoint and debug as you wish, maybe you can try `CTRL+N` to find `TestOperator` to start your test
+- Make sure you have compiled gluten.
+- Load the gluten by File->Open, select <gluten_home/pom.xml>.
+- Activate your profiles such as <backends-velox>, and reload Maven Project, you will find all your need modules have been activated.
```

Comment:
This `Reload` is the button description in IDEA

Reply:
OK